kyo-core/shared/src/main/scala/kyo/internal/Position.scala

```diff
@@ -0,0 +1,32 @@
+package kyo.internal
+
+import scala.annotation.tailrec
+import scala.quoted.*
+
+opaque type Position = String
+
+object Position:
```

Comment:
Renamed `FlatImplicits` to `Position` and made a few cleanups. I think git didn't detect the rename because of the changes.

Reply:
If you start with `git mv`, it should detect correctly and maintain the blame.